geode-for-redis/src/main/java/org/apache/geode/redis/internal/commands/executor/list/LInsertExecutor.java

```diff
@@ -39,10 +39,10 @@ public RedisResponse executeCommand(Command command, ExecutionHandlerContext con
     boolean before;
     byte[] referenceElement = commandElements.get(3);
     byte[] elementToInsert = commandElements.get(4);
-
-    if (Arrays.equals(direction, BEFORE)) {
+    String directionInUppercase = new String(direction).toUpperCase();
```

Comment:
Rather than creating a new String here, line 38 should be 
```
byte[] direction = toUpperCaseBytes(commandElements.get(2));
```
The whole point of doing the byte array comparison with the constants from `StringBytesGlossary` is to avoid having to create a String.